### PR TITLE
chore(api): make SlugsUpdateEndpoint private

### PR DESCRIPTION
**Pull request by michellewzhang**
image

fixed the typo, screenshot is from pre-fix

### CODE REVIEW

File: src/sentry/api/endpoints/organization_slugs.py [hunk 1]
```diff
@@ -1,34 +1,53 @@
...
 @region_silo_endpoint
 class SlugsUpdateEndpoint(OrganizationEndpoint):
+    owner = ApiOwner.UNOWNED
     publish_status = {
-        "PUT": ApiPublishStatus.UNKNOWN,
+        "PUT": ApiPublishStatus.PUBLIC,
     }
 
+    @extend_schema(
+        operation_id="Update an Origanization's Project Slugs",
```
Comment (isabellaenriquez) [inline src/sentry/api/endpoints/organization_slugs.py]:
```suggestion
        operation_id="Update an Organization's Project Slugs",
```

Comment (JoshFerge) [inline src/sentry/api/endpoints/organization_slugs.py:31]:
```suggestion
        operation_id="Update an Organization's Project Slugs",
```

File: src/sentry/api/endpoints/organization_slugs.py [hunk 2]
```diff
@@ -1,34 +1,53 @@
...
+        parameters=[GlobalParams.ORG_ID_OR_SLUG],
+        request=inline_serializer(
+            name="UpdateOrgProjectSlugs",
+            fields={
+                "slugs": serializers.DictField(
+                    help_text="a dictionary of project IDs to their intended slugs.", required=False
```
Comment (isabellaenriquez) [inline src/sentry/api/endpoints/organization_slugs.py:37]:
```suggestion
                    help_text="A dictionary of project IDs to their intended slugs.", required=False
```

could be helpful to include an example as well

**Review by JoshFerge — COMMENTED** [review]
are we sure we want to publish this one? i feel like the project details PUT might be sufficient for doing something like this https://docs.sentry.io/api/projects/update-a-project/

File: src/sentry/api/endpoints/organization_slugs.py [hunk 3]
```diff
@@ -1,34 +1,53 @@
...
+                ),
+            },
+        ),
+        responses={
+            200: inline_sentry_response_serializer("SlugsUpdateResponse", list[str]),
```
Comment (isabellaenriquez) [inline src/sentry/api/endpoints/organization_slugs.py:42]: i think the response type should be dict?

Comment (michellewzhang) [reply src/sentry/api/endpoints/organization_slugs.py]: oh yes good catch!

**Comment by michellewzhang** [comment]
> are we sure we want to publish this one? i feel like the project details PUT might be sufficient for doing something like this https://docs.sentry.io/api/projects/update-a-project/

hmm you could be right @JoshFerge  - cc @sentaur-athena, what do you think?

**Comment by JoshFerge** [comment]
> > are we sure we want to publish this one? i feel like the project details PUT might be sufficient for doing something like this https://docs.sentry.io/api/projects/update-a-project/
> 
> hmm you could be right @JoshFerge - cc @sentaur-athena, what do you think?

yeah i feel strongly that we shouldn't publish this one -- this seems like an endpoint that is useful for the frontend but doesn't seem necessary for a list of backend APIs as it feels redundant and confusing compared to the project PUT update which can update slugs

**Review by isabellaenriquez — APPROVED** [review]

**Pull request merged**